dc-cudami-server/dc-cudami-server-backend-jdbi/src/main/java/de/digitalcollections/cudami/server/backend/impl/jdbi/identifiable/entity/DigitalObjectRepositoryImpl.java

```diff
@@ -678,4 +688,16 @@ protected String getColumnName(String modelProperty) {
         return null;
     }
   }
+
+  @Override
+  public boolean deleteIdentifiers(UUID digitalObjectUuid) {
+    DigitalObject digitalObject = findOne(digitalObjectUuid);
+    if (digitalObject == null) {
+      return false;
+    }
+
+    digitalObject.getIdentifiers().stream().forEach(i -> identifierRepository.delete(i.getUuid()));
```

Comment:
y tho? :-( `DELETE FROM bla WHERE uuid IN ?, ?, ?, ?` und das ganze ist innerhalb einer einzigen DB-Aktion gemacht.

Reply:
vielen Dank, ist umgebaut